configure.ac

```diff
@@ -275,6 +275,21 @@ AS_IF([test "$enable_check_asm" = "yes"], [
   AC_DEFINE([OD_CHECKASM], [1], [Validate assembly code])
 ])
 
+AC_ARG_ENABLE([mc-animate],
+  AS_HELP_STRING([--enable-mc-animate], [Animate mc]),,
```

Comment:
Please document the use of the ANIFRAME here (in the [Animate MC] text), so that someone reading ./configure --help has a chance o fusing it correctly.